src/template-tags/tickets.php

```diff
@@ -221,26 +222,37 @@ function tribe_tickets_get_ticket_stock_message( Tribe__Tickets__Ticket_Object $
 			$stock = '<i>' . __( 'global inventory', 'event-tickets' ) . '</i>';
 		}
 
+		$sold_label = __( 'Sold', 'event-tickets' );
+		if ( 'Tribe__Tickets__RSVP' === $ticket->provider_class ) {
+			$sold_label = __( 'RSVP\'d Going', 'separate going and remain RSVPs', 'event-tickets' );
+		}
+
 		// There may not be a fixed inventory - in which case just report the number actually sold so far
 		if ( empty( $stock ) && $stock !== 0 ) {
-			$message = sprintf( esc_html__( 'Sold %d', 'event-tickets' ), esc_html( $sold ) );
-		}
-		// If we do have a fixed stock then we can provide more information
+			$message = sprintf( esc_html__( '%s %d', 'event-tickets' ), esc_html( $sold_label ), esc_html( $sold ) );
+		} // If we do have a fixed stock then we can provide more information
 		else {
-			$cancelled_count = empty( $cancelled ) ? '' : esc_html( sprintf(
-				_x( ' cancelled: %1$d', 'ticket stock message (cancelled stock)', 'event-tickets' ),
-				(int) $cancelled
-			) );
-
-			$pending_count = $pending < 1 ? '' : esc_html( sprintf(
-				__( ' pending: %1$d', 'ticket stock message (pending stock)', 'event-tickets' ),
-				(int) $pending
-			) );
-
-			$message = sprintf(
-				esc_html__( 'Sold %1$d (units remaining: %2$s%3$s%4$s)', 'event-tickets' ),
-				esc_html( $sold ), $stock, $cancelled_count, $pending_count
-			);
+			$status = '';
+
+			if ( $is_global && 0 < $stock && $global_stock->is_enabled() ) {
+				$status_counts[] = esc_html( sprintf( __( '%1$d Remaining of the global stock', 'ticket global stock message (remaining stock)', 'event-tickets' ), (int) $stock ) );
```

Comment:
Here and quite a few of the following lines, we seem to be providing context for the translations but we're using a straight `__()` - we need to fix and either drop the context or move to `_x()` or similar :+1: